tests/api/runtimes/test_pod.py

```diff
@@ -50,3 +50,24 @@ def test_with_limits_regex_validation(self):
                 )
                 if not case.get("gpus"):
                     kube_resource.with_requests(case.get("memory"), case.get("cpu"))
+
+    def test_function_with_handler_function(self):
+        """
+        Some users write their code in a notebook, then do code_to_function, and then mistakenly give the function
+        itself to the handler kwarg, instead of giving the function name (string)
+        This test is here to verify that the failure is fast and clear
+        """
+
+        def some_func():
+            pass
+
+        with pytest.raises(
+            mlrun.errors.MLRunInvalidArgumentError,
+            match=r"Parameter default_handler must be a string. got <class 'function'>",
```

Comment:
Same as last comment.
```suggestion
            match=r"Parameter default_handler must be a string. Got <class 'function'>",
```